test/fuzz/transformation_replace_load_store_with_copy_memory_test.cpp

```diff
@@ -186,9 +255,23 @@ TEST(TransformationReplaceLoadStoreWithCopyMemoryTest, BasicScenarios) {
                OpCopyMemory %14 %12
          %24 = OpLoad %16 %18
                OpCopyMemory %20 %18
+               OpStore %26 %27
+               OpStore %28 %27
+         %29 = OpLoad %6 %26
+               OpMemoryBarrier %32 %32
+               OpStore %28 %29
+               OpStore %31 %32
+               OpStore %33 %32
+         %34 = OpLoad %6 %33
+               OpSelectionMerge %38 None
+               OpBranchConditional %36 %37 %38
+         %37 = OpLabel
+               OpStore %31 %34
+               OpBranch %38
+         %38 = OpLabel
                OpReturn
                OpFunctionEnd
-  )";
+    )";
   ASSERT_TRUE(IsEqual(env, after_transformations, context.get()));
```

Comment:
It would be good to assert the reference shader is valid after the transformation.
You can use `ASSERT_TRUE(IsValid(env, context.get()));` here.

Reply:
I have this check after each applied transformation. See line: 186, 193

Reply:
You're right.
Sorry for the confusion.